steembase/http_client.py

```diff
@@ -1,20 +1,24 @@
 # coding=utf-8
-import concurrent.futures
 import json
 import logging
 import socket
 import time
+import sys
 from functools import partial
-from http.client import RemoteDisconnected
 from itertools import cycle
-from urllib.parse import urlparse
-
+import concurrent.futures
 import certifi
 import urllib3
 from steembase.exceptions import RPCError
 from urllib3.connection import HTTPConnection
 from urllib3.exceptions import MaxRetryError, ReadTimeoutError, ProtocolError
 
+if sys.version >= '3.0':
+    from http.client import RemoteDisconnected
+else:
+    from urlparse import urlparse
```

Comment:
I noticed in one of the last commits this change:
```
  if sys.version >= '3.0':
-    from urllib.parse import urlparse
```
Isn't this import still needed for python3?

Reply:
Yup! Not sure how that got removed.